plugins/pur/core/src/test/java/org/pentaho/di/repository/pur/PurRepository_DatabaseNames_Test.java

```diff
@@ -71,7 +71,7 @@ public void setUp() throws Exception {
     purRepository.setTest( unifiedRepository );
   }
 
-  @Test
+  //@Test
```

Comment:
we're uncommenting this eventually, right?

Reply:
Yes, eventually